### PR TITLE
Fix loading across app

### PR DESCRIPTION
**Pull request by thejoefriel**
This adds styled loading icons to all relevant pages and tables across the app. 

For some reason sometimes the Table doesn't want to re-render so have had to write different ways sometimes of getting it to work. I think this is to do with the reducers and how it gets the data. I'm going to create an issue for us to refactor this at some point for stronger consistency in the codebase.

### CODE REVIEW

File: client/src/pages/Staff/Members/index.js [hunk 1]
```diff
@@ -85,6 +86,8 @@ const Transactions = ({
     "Time credits SPENT",
     "% Time credits spent",
   ];
+
+  console.log("in", insightsLoading);
```
Comment (dupreesi) [inline client/src/pages/Staff/Members/index.js:90]: commented code

File: client/src/pages/Dashboard/Member/Header.js [hunk 2]
```diff
@@ -2,26 +2,22 @@ import React from "react";
...
-        credits={userTc || "N/A"}
-        memberID={membershipId}
-        type="digital"
-      />
-    );
-  };
+  const renderMemberCard = (width) => (
+    <CreditCard
+      width={width}
+      credits={userTc || "N/A"}
```
Comment (dupreesi) [inline client/src/pages/Dashboard/Member/Header.js:15]: maybe give it "0" if no userTC? I

File: client/src/pages/Staff/Members/index.js (hunk 1 above)
Comment (dupreesi) [inline client/src/pages/Staff/Members/index.js:90]: commented code

Comment (thejoefriel) [reply client/src/pages/Staff/Members/index.js:90]: done

File: client/src/pages/Dashboard/Member/Header.js (hunk 2 above)
Comment (thejoefriel) [reply client/src/pages/Dashboard/Member/Header.js:15]: good idea, sorted

File: client/src/pages/Staff/Members/index.js (hunk 1 above)
Comment (thejoefriel) [reply client/src/pages/Staff/Members/index.js:90]: same as above so sorted :)

**Pull request merged**